### DISCUSSION TITLE
Global Campus Verification doesn't take teacher email address

### DISCUSSION THREAD

**Discussion by kumkee** [community/community#50909] [GitHub Education]
> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Mar 23, 2023 for @kumkee.

Excuse me? I am applying through my school email address ending school's domain name (macleans.school.nz)

> Attendees of this school must verify and use their academic email address.

Excuse me? I am trying to get verified as a teacher not an attendee. Your system only accept email address with student subdomain of our school (student.macleans.school.nz). Of course I don't have one!

Can I get a real person to verify my case please?

**Comment by samus-aran** [comment]
Hi @kumkee 👋 

Thanks for bringing this to our attention. We've passed this along to our internal team and any updates we'll post here. 👍

**Reply by kumkee** [reply]
Finally, I hear from a fellow human. Thank you @samus-aran. 
Yes, please keep me informed of your progress and let me know if I need to provide any additional documents from the school.

**Reply by samus-aran** [reply]
Hi @kumkee 

We have updated the details of your school. We please ask you to re-apply and let us know how you get on 👍

**Reply by kumkee** [reply]
> 

Thank you, @samus-aran. I am verified and now a member of GitHub Global Campus 👏 
I am not sure if I shall close this discussion so I'll leave that to you.

**Reply by samus-aran** [reply]
That's awesome @kumkee ! 🎉 Thanks so much for coming back and sharing the update. This will help other community members seeking similar answers. Start enjoying GitHub Global Campus 🌟

As a verified Teacher you are also invited to the [private Global Campus Teachers Discussion boards](https://github.com/community/Global-Campus-Teachers/discussions) ✅ 

It's great to have you in the GitHub Education community and we hope to see you around 👋
